recipe/meta.yaml

```diff
@@ -54,6 +54,7 @@ requirements:
     - libnetcdf
     - udunits2
     - tempest-remap  # [not win]
+    - moab * tempest_*  # [not win]
```

Comment:
@czender, can you confirm that we want `moab` with `tempest` to always be a dependency of `nco`?  Or do we want it to be optional, in which case it should be installed for testing only.

Reply:
Based on the below, can we just leave MOAB to be installed separately from NCO? So do not make MOAB an NCO dependency? That's how I've been testing it and it works fine, at least in an MPI environement.

Reply:
Okay, so we build without requiring `moab` or MPI but test with the `mpi_mpich_tempest_*` build of `moab`.  That sounds like it will work.